railgun/internal/proxy/clientgo_cached_proxy_resolver.go

```diff
@@ -60,8 +61,9 @@ func NewCacheBasedProxyWithStagger(ctx context.Context, logger logr.Logger, k8s
 // This object's attributes are immutable (private), and it is threadsafe.
 type clientgoCachedProxyResolver struct {
 	// kubernetes configuration
-	k8s   client.Client
-	cache *store.CacheStores
+	k8s       client.Client
+	cache     *store.CacheStores
+	updateSHA []byte
```

Comment:
I think `lastConfigSHA` would be a better name and I'd add a docstring:

```go
// lastConfigSHA represents a checksum of the last configuration applied to Kong Admin API.
```

Reply:
:+1: 8db2228638a070f1403caeab7fa8304504f08f7d